template/index.ts

```diff
@@ -18,16 +23,16 @@ const userSchema = new Schema({
   `;
 }
 
-function schemaBottom() {
+export function schemaBottom() {
   return `
   timestamps: { createdAt: "created_at", updatedAt: "updated_at" } 
 });
 module.exports = userSchema;
   `;
 }
 
-module.exports = {
-  createSchemaObject,
-  schemaTop,
-  schemaBottom,
-};
+// module.exports = {
```

Comment:
No point holding onto this! Now that exports are done using ES6 modules, you can get rid of these CommonJS artifacts